packages/react/src/components/Comments/FloatingComposerController.tsx

```diff
@@ -32,17 +32,18 @@ export const FloatingComposerController = <
   }
 
   const comments = editor.comments;
+  useEffect(() => {
```

Comment:
was this a code style preference to go from:

```
useEffect(() => {	
    editor.ForceSelectionVisible = !!state?.pendingComment;	
  }, [editor, state?.pendingComment]);
```
to 

```
 useEffect(() => {
comments.onUpdate((state) =>
      editor.setForceSelectionVisible(state.pendingComment)
    );
  }, [comments, editor]);
```

or does it solve a bug?

In any case, we're not unsubscribing from `onUpdate` so that would need to be fixed I think